docs/sources/articles/dockerfile_best-practices.md

```diff
@@ -335,12 +331,31 @@ fi
 exec "$@"
 ```
 
-That script then gets copied into the container and run via `ENTRYPOINT` on
-container startup:
+> **Note**: This script uses [the `exec` Bash command](http://wiki.bash-hackers.org/commands/builtin/exec)
+> so that the final running application becomes the container's `PID 1` so that
+> it will receive any Unix signals sent to the container. See the
```

Comment:
Mmm hmm, I think there is more stress-worthy material in this one sentence than there are stress positions available so.... let's break it into two sentences:
"This script uses [the `exec` Bash command](http://wiki.bashhackers.org/commands/builtin/exec)
so that the final running application becomes the container's `PID 1`. This allows the application to receive any Unix signals sent to the container."